iaso/tasks/profiles_bulk_update.py

```diff
@@ -32,21 +39,41 @@ def update_single_profile_from_bulk(
     language: Optional[str],
 ):
     """Used within the context of a bulk operation"""
-    original_copy = deepcopy(profile)
+    # original_copy = deepcopy(profile)
+    audit_logger = ProfileAuditLogger()
+    old_data = audit_logger.serialize_instance(profile)
     account_id = user.iaso_profile.account.id
+    roles_to_be_added = []
+    roles_to_be_removed = []
+    projects_to_be_added = []
+    projects_to_be_removed = []
+    org_units_to_be_added = []
+    org_units_to_be_removed = []
+    # Raise if necessary and prepare values to be updated
+    if teams_id_added is not None:
```

Comment:
The thing is I want to raise all exceptions before writing to the DB to avoid having partial DB updates and no logs for them. That's why the code is split